lib/gracenote/response/cover_art.rb

```diff
@@ -0,0 +1,21 @@
+require "gracenote/response/base"
+
+module Gracenote
+  class Response
+    class CoverArt < Base
+      self.attributes = %w[type size width height]
```

Comment:
__[rubocop]__
* `%w`-literals should be delimited by `(` and `)`. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)